test/server/core/abilities/keyword/Piloting.spec.ts

```diff
@@ -261,6 +261,33 @@ describe('Piloting keyword', function() {
                 expect(context.concordDawnInterceptors).toBeInZone('hand');
                 expect(context.idenVersio).toBeInZone('discard');
             });
+
+            it('can be moved to another vehicle with a Pilot ignoring the limit', async function () {
```

Comment:
This has been confirmed [here](https://bsky.app/profile/davflamerock.bsky.social/post/3li3jgzevck27).